### PR TITLE
[WIP] Updated CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by agarciavera**
- Updated CONTRIBUTING.md for good contributing.
- Minor fix in README.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -12,6 +12,21 @@ Opening issues will allows us to write better code and make sure it is more usab
 
 If you wish to contribute with a translation of the project in your language, we are of course happy to accept them.
 
+## Steps to contribute
+We are using test to check the code before merging. If you wish open a pull request to fix any bug o enhancement the application,
```
Comment (sergio-ocon) [inline CONTRIBUTING.md]: "All PR need to pass some tests before they are merged. You can find a list of tests in travis-ci.org.
You can also run the same tests locally to make sure that PR can be accepted. In order to do so, you can execute the following commands"

'Bundle install' installs the gems, there is no need to refer to that.

File: README.md [hunk 2]
```diff
@@ -65,7 +65,7 @@ $ rails s              # Application starting in development
 ```
 - Using your favorite browser, go to [http://localhost:3000] (http://localhost:3000)
 
-- You can also install dependencies and database manually. If you choose this way, you should install `bower` component before starting application.
+You can also install dependencies and database manually. If you choose this way, you should install `bower` component before starting application.
```
Comment (sergio-ocon) [inline README.md]: 'bower' components?

**Comment by agarciavera** [comment]
I think that now everything is solved @sergio-ocon.

**Comment by sergio-ocon** [comment]
The do a follow up PR

**Pull request merged**